build.gradle

```diff
@@ -135,3 +166,11 @@ task ktlint(type: JavaExec, group: "verification") {
     main = "com.github.shyiko.ktlint.Main"
     args "app/**/*.kt"
 }
+
+
+task listRepositories {
```

Comment:
Kudos to this stackoverflow post https://stackoverflow.com/questions/32143437/how-to-list-the-configured-repositories